### PR TITLE
Feature/startup screen

### PR DESCRIPTION
**Pull request by hrantzsch**
* add drawing.py with a few predefined letters and figures
* ball now contains it's own surface
* show 'P0' and 'P1' in gray until players join, turn green when the respective player has joined

### CODE REVIEW

File: pong.py [hunk 1]
```diff
@@ -23,31 +25,53 @@ def __init__(self, mlhost, mlport, screen_width, screen_height):
...
+            p0_color = pymlgame.GREEN
+        if len(self.players) > 1:
+            p1_color = pymlgame.GREEN
+
+        surface = pymlgame.Surface(self.screen.width, self.screen.height)
+        # note(hrantzsch): this works for our default screen size only
+        drawing.draw_p(surface, Point(0, 2), p0_color)
+        drawing.draw_0(surface, Point(4, 2), p0_color)
+        drawing.draw_p(surface, Point(10, 2), p1_color)
+        drawing.draw_1(surface, Point(14, 2), p1_color)
```
Comment (johannesjasper) [inline pong.py:68]: A lot of magic numbers flying around here. Ideally, you would have something like a  `wating_screen` class, which holds its own surface and the reference to `drawing`.

File: pong.py [hunk 2]
```diff
@@ -23,31 +25,53 @@ def __init__(self, mlhost, mlport, screen_width, screen_height):
...
 
+        self.screen.update()
+        self.clock.tick()
+
+    def update_waiting(self):
+        if len(self.players) == 2:
+            self.state = "START_GAME"
+
+    def render_waiting(self):
+        p0_color = pymlgame.GREY5
```
Comment (johannesjasper) [inline pong.py]: set either grey, or green, rather than overwriting it. use inline `if..else` or ternary.

**Review by johannesjasper — APPROVED** [review]
Looks good! I added some nitpicky comments ;)

**Comment by Bartzi** [comment]
Quite neat ;)
I think we should use something like `pygame.font` for text rendering http://stackoverflow.com/questions/10077644/python-display-text-w-font-color
I'm not sure how well it works with pymlgame, but should not be too hard to integrate it^^

File: pong.py (hunk 2 above)
Comment (hrantzsch) [reply pong.py]: will do

**Comment by hrantzsch** [comment]
@Bartzi Sound's good, but maybe we can look into this later?

**Comment by Bartzi** [comment]
Sure^^ was just an idea I had... I will open a new issue for that!
Merge once you are ready^^

**Review by Bartzi — APPROVED** [review]

**Pull request merged**